core/src/main/java/org/frankframework/jms/JmsSender.java

```diff
@@ -310,7 +310,7 @@ private void setProperties(javax.jms.Message msg, ParameterValueList pvl) throws
 			ParameterType type = property.getDefinition().getType();
 			String name = property.getDefinition().getName();
 
-			if ((!isSoap() || !name.equals(getSoapHeaderParam()) && (StringUtils.isEmpty(getDestinationParam()) || !name.equals(getDestinationParam())))) {
```

Comment:
Hier vind ik hem wel echt handig!

Reply:
Het moet volgens mij zijn:
`(!isSoap() || !name.equals(getSoapHeaderParam()))`
EN
`(StringUtils.isEmpty(getDestinationParam()) || !name.equals(getDestinationParam()))`

Reply:
Volgens Vivy is dit goed: `if (!isSoap() || !name.equals(getSoapHeaderParam()) && !name.equals(getDestinationParam())) {`

Reply:
Aangepast. Dank beiden.